requirements.txt

```diff
@@ -4,8 +4,16 @@ pytest-cov
 pytest-runner
 build
 bumpversion
-freezegun
-pandas
-selenium
+freezegun~=1.2.2
+pandas~=2.0.1
+selenium~=4.9.1
 webdriver-manager
-sphinx
\ No newline at end of file
+sphinx
+requests~=2.28.1
```

Comment:
Do we actually need any of these? We need freezegun and sphinx for tests and documentation respectively but the setup.cfg file is where the install lives

Reply:
done